### PR TITLE
Added code to change password of hscroot user

### PR DESCRIPTION
**Pull request by SreenidhiS1**

### CODE REVIEW

File: plugins/modules/hmc_user.py [hunk 1]
```diff
@@ -737,7 +743,15 @@ def ensure_update(module, params):
                     filter_d['NAMES'] = attributes['new_name']
                 changed = True
 
+        if m_config["NAME"] == 'hscroot' and m_config["passwd"] is not None and hmc_user == 'hscroot':
```
Comment (naveenkandakur) [inline plugins/modules/hmc_user.py]: define a variable and assign hscroot string to it and use that variable wherever you are hardcoding

File: plugins/modules/hmc_user.py [hunk 2]
```diff
@@ -737,7 +743,15 @@ def ensure_update(module, params):
...
+            password = m_config["passwd"]
+            hmc_conn = HmcCliConnection(module, hmc_host, hmc_user, password)
```
Comment (naveenkandakur) [inline plugins/modules/hmc_user.py:750]: Please add a comment why we are re-initializing the HmcCliConnection?

**Review by naveenkandakur — CHANGES_REQUESTED** [review]

File: plugins/modules/hmc_user.py (hunk 1 above)
Comment (SreenidhiS1) [reply plugins/modules/hmc_user.py]: Done

File: plugins/modules/hmc_user.py (hunk 2 above)
Comment (SreenidhiS1) [reply plugins/modules/hmc_user.py:750]: Done

**Review by naveenkandakur — APPROVED** [review]

**Pull request merged**